Solutions/PCMatic SuperShield/Logo/pcmatic-green-outline.svg

```diff
@@ -0,0 +1 @@
13,3.07,3.52-1.19,5.34-4.81,6.74-8,2.69-6.51,4.1-13.44,5.85-20.24,9.75,0,19.51,0,29.27,0-1.8,6.38-2.75,13-5.22,19.18-4.36,12-13.38,22.81-25.59,27.24-9.81,3.36-20.47,3.63-30.65,2.1-6.8-1.13-13.84-4.3-17.23-10.63-4.18-7.88-3.58-17.17-2.69-25.75a253.07,253.07,0,0,1,6.67-33.4c3.41-12.22,7.29-24.74,15.11-34.94a39,39,0,0,1,23.19-14.75m-108.07.74q23.22,0,46.44,0c7.45.12,15.39,2.41,20.28,8.38C260.53,56,260.9,65,259.7,72.88c-2,13.91-8.91,28.09-21.69,35-10.85,6.39-23.8,6.11-36,6-3.42,14.87-6.7,29.77-10.22,44.62-10,0-20.08-.06-30.13,0q13.4-58.65,26.75-117.3m25.15,22c-2.22,9.53-4.33,19.09-6.55,28.63,4.4,0,9.2.47,13.12-2,4.93-3,6.88-8.87,7.9-14.22.61-3.64.75-8.24-2.53-10.7s-8-1.56-11.94-1.73M392.41,41.31c14.44-.22,28.88,0,43.32-.1-1.46,23.33-2.23,46.7-4,70,1.94-2.91,2.83-6.31,4.18-9.5,8.18-20.16,16.2-40.39,24.46-60.53,14.39.07,28.78,0,43.18,0-9.2,39-17.86,78.18-26.86,117.25q-14.14,0-28.3,0c6.18-26.17,11.95-52.44,18-78.63.8-2.88,1.12-6,2.64-8.59-1-.27-1.56.73-1.79,1.58-11.6,28.61-23.44,57.12-35.2,85.67-7.27-.09-14.55.11-21.83-.11,1.54-29.19,2.68-58.41,4-87.6-1.48,3.3-1.77,7-2.62,10.44Q402.85,119.88,394,158.5c-9.42,0-18.84,0-28.26,0,8.94-39.05,18-78.08,26.64-117.18m243.3-.05c9.76-.11,19.52,0,29.28-.06-1.77,7.34-3.33,14.74-5.13,22.08-9.73-.08-19.46,0-29.19,0q2.63-11,5-22M586,69.63q3-13,5.94-26c9.75,0,19.51,0,29.26,0-2.18,8.64-3.85,17.4-6.1,26,3.88.09,7.77.07,11.66,0-1.62,6.6-3,13.25-4.58,19.86-3.83,0-7.66,0-11.48,0-3.37,14.79-6.86,29.55-10.05,44.38-.72,5.79,6.57,5.07,10.33,5.07-1.56,6.49-3.07,13-4.47,19.53a109.53,109.53,0,0,1-26,.51c-3.85-.39-8.14-1.94-9.9-5.67-2.4-5.08-1-10.86,0-16.11,3.66-15.9,7.16-31.84,11-47.71-3.29,0-6.58,0-9.87,0,1.61-6.55,3-13.16,4.57-19.73,3.23-.17,6.48-.12,9.73-.11m-76,7.15c10.54-9.27,25.66-10.49,39-8.69,6.84.71,14.48,3.53,17.41,10.29,3.29,7.34.86,15.44-.8,22.86-3,13.44-6.15,26.87-9.17,40.32-1.06,5.58-2.61,11.32-1.24,17-9.41-.09-18.81,0-28.21-.09a50.24,50.24,0,0,1,.72-10c-4.4,5.73-10.4,10.52-17.68,11.81s-15.61.69-20.94-4.82c-4.42-4.72-4.35-11.7-3.41-17.68,1.63-9.14,5.11-18.84,13.09-24.31,10-7.05,22.68-7.56,34-11.16,5.77-1.84,9-8.9,7.07-14.5-2.24-3.16-6.94-2.78-10.07-1.34-4.08,2-5,6.91-6.32,10.84-8.47-.12-16.94,0-25.4-.08,2-7.75,5.63-15.29,11.91-20.49m25.39,38.65c-3.8,3.27-9.26,2.6-13.35,5.35-4.59,2.56-6.47,7.94-7.14,12.86-.52,2.92.85,6.62,4,7.31a10,10,0,0,0,10.19-5c3.68-6.24,4.09-13.71,6.28-20.47M674,75.81c8.56-7.18,20.21-8.41,31-8.44,8.09.13,17.38,2,22.17,9.23,5.36,8.36,3.24,18.75,1.52,27.88-8.82.06-17.65.15-26.47,0,1.14-4.74,2.38-9.77,1.28-14.62-1.53-4.36-7.77-4.11-10.77-1.46-3.54,3.41-4.82,8.39-6.32,12.91a211.32,211.32,0,0,0-6.52,30.45c-.16,3.08-.66,7,2,9.17,4,2.31,9.29.1,11.55-3.68,2.88-4.75,3.71-10.37,4.9-15.7,8.81-.14,17.62,0,26.44-.09-2.87,11.38-6.54,23.65-16.09,31.26-11,8.64-25.94,9.18-39.19,7.31-6.66-.81-13.35-4.44-16-10.86-4.66-10.79-2.19-22.88.19-33.91,3.33-14.45,8.61-29.6,20.35-39.4m-44.72-6.13c9.75-.13,19.5,0,29.25-.07-6.88,29.61-13.51,59.27-20.33,88.88-9.76,0-19.52,0-29.28,0,7-29.54,13.52-59.2,20.36-88.78Z"/></g></g></svg>
```

Comment:
@johnpcmatic, Do not use illegal attributes: data-name, style, xmlns:link and SVG Id should be unique guid, thanks.